src/pybind/mgr/rook/module.py

```diff
@@ -134,6 +134,25 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class RookEnv(object):
+    def __init__(self):
+        # POD_NAMESPACE already exist for Rook 0.9
+        pod_namespace = os.environ.get('POD_NAMESPACE', 'rook-ceph')
+        self.cluster_ns = os.environ.get('ROOK_CLUSTER_NS', pod_namespace)
```

Comment:
These are the env vars added in https://github.com/rook/rook/pull/2903/files:
```go
		{Name: "ROOK_OPERATOR_NAMESPACE", Value: operatorNamespace},
		{Name: "ROOK_CEPH_CLUSTER_CRD_VERSION", Value: rookcephv1.Version},
		{Name: "ROOK_VERSION", Value: rookversion.Version},
		{Name: "ROOK_CEPH_CLUSTER_CRD_NAME", Value: c.clusterInfo.Name},
```
`POD_NAMESPACE` also exists in Rook, but I don't see mention of `ROOK_CLUSTER_NS` in Rook's code base. Should this be removed here?

As a related note, I have a strong preference to never use shorthand (i.e. `NS` instead of `NAMESPACE`) for variables like these that transfer information from Rook to Ceph. This effectively represents an API between two entirely different projects, and eliminating ambiguity at boundaries like this should be a requirement.

Reply:
At some point in the past, there was a `ROOK_CLUSTER_NS` env var. If Rook no longer sets it, it won't hurt anyone.

Reply:
can we use same style for variables, like ROOK_CLUSTER_NAMESPACE and POD_NAMESPACE, and ROOK_CLUSTER_NS is confusing, since NS is usually abbreviation for nameserver.

Reply:
@kshtsk I'd like to not introduce any non-critical changes in this PR

Reply:
I can confirm that recent Rook masters do not set the `ROOK_CLUSTER_NS` variable according to `env`. The four I mentioned above are present.

Reply:
So, there are two possibilities for this backport PR: Either we merge this PR or we decline it. Keep in mind that this PR is already merged into `master` and introducing changes between `master` and `nautilus` is nothing I'd want to do.

Reply:
Add one more patch to master fixing the issue, then cherry-pick -x that into this branch.

Reply:
See #27556 for renaming the vars